### PR TITLE
Send a notification whenever data is updated

### PR DESCRIPTION
**Pull request by theramis**
it's getting annoying checking github if data has been updated. 

This will send us a message on discord when the data is updated.

### CODE REVIEW

**Review by andycarrell — APPROVED** [review]
Let's merge this in - will leave some comments that you can address now or later ;)

File: shared/dataRepository.ts [hunk 1]
```diff
@@ -26,18 +26,11 @@ export const getVaccineDataForDate = async (
...
-    return;
-  }
-
-  if (areObjectsDifferent(existingData, data)) {
-    await createVaccineDataForDate(time, data);
-  }
+  await createVaccineDataForDate(time, data);
 };
 
 const areObjectsDifferent = (a: object, b: object) =>
```
Comment (andycarrell) [inline shared/dataRepository.ts:36]: Is this function still used in this file?

File: shared/dataRepository.ts [hunk 2]
```diff
@@ -26,18 +26,11 @@ export const getVaccineDataForDate = async (
...
 ) => {
-  const existingData = await getVaccineDataForDate(time);
-  if (existingData == null) {
-    return;
-  }
-
-  if (areObjectsDifferent(existingData, data)) {
-    await createVaccineDataForDate(time, data);
-  }
+  await createVaccineDataForDate(time, data);
```
Comment (andycarrell) [inline shared/dataRepository.ts:33]: This is pass through, so, is there any difference between `createVaccineDataForDate` & `updateVaccineDataForDate`? If not can we just use `createVaccineDataForDate`?

File: fetchy/fetchyboi.ts [hunk 3]
```diff
@@ -2,40 +2,62 @@ import getVaccineData from "./vaccineDataService";
...
+};
+
+const sendDataUpdatedNotification = async () => {
+  await sendNotification(
+    ":raised_hands: :raised_hands: **Fetchy got some new data** :raised_hands: :raised_hands:"
+  );
+};
+
+const areObjectsDifferent = (a: object, b: object) =>
+  Object.keys(diff(a, b)).length !== 0;
```
Comment (andycarrell) [inline fetchy/fetchyboi.ts:33]:
```suggestion
  Boolean(Object.keys(diff(a, b)).length);
```

File: fetchy/fetchyboi.ts [hunk 4]
```diff
@@ -2,40 +2,62 @@ import getVaccineData from "./vaccineDataService";
...
   getVaccineDataForDate,
-  updateVaccineDataIfDifferentForDate,
+  updateVaccineDataForDate,
 } from "../shared/dataRepository";
 import got from "got";
+import { diff } from "deep-object-diff";
+
+const sendNotification = async (message: string) => {
+  // don't send notifications if running in a PR
+  if (process.env.GITHUB_EVENT_NAME === "pull_request") {
```
Comment (andycarrell) [inline fetchy/fetchyboi.ts:12]: Does this send notifications if you run `fetchyboi` locally? Maybe we should make this an opt in to messaging and then we set an env var in the github action?

```suggestion
  if (process.env.SEND_DISCORD_NOTIFICATIONS === "true") {
```

Comment (theramis) [reply fetchy/fetchyboi.ts:12]: Ooo yes much better idea. Will do in another pr

**Pull request merged**